### DISCUSSION TITLE
"Decasking"

### DISCUSSION THREAD

**Discussion by alexreg** [Homebrew/discussions#1891] [Casks] [answered]
Is there any way to "decask" an installed cask? That is, have Homebrew no longer manage a particular cask/app, but leave it installed on the system. If not, this would certainly be a handy feature.

**Comment by gromgit** [comment, marked as answer]
I believe it's as simple as:
```
rm -fr ${HOMEBREW_PREFIX}/Caskroom/<cask_name>
```
Note that some casks do additional stuff like adding symlinks to internal binaries (e.g. `mkvtoolnix`). Not sure what your use case for "decasking" is, but if you want to undo those too, there's no easy way to do that.

**Reply by alexreg** [reply]
@gromgit Thanks, that does the job for me. For my purposes, the symlinking of internal binaries isn't a real issue.
